esx_service/cli/vmdkops_admin.py

```diff
@@ -974,8 +981,11 @@ def tenant_ls(args):
         return operation_fail(error_info.msg)
 
     header = tenant_ls_headers()
-    rows = generate_tenant_ls_rows(tenant_list)
-    print(cli_table.create(header, rows))
+    error_info, rows = generate_tenant_ls_rows(tenant_list)
+    if error_info:
+        print error_info.msg
```

Comment:
root@dhcp-192-168-31-62:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls
  File "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py", line 986
    print error_info.msg
                   ^
SyntaxError: Missing parentheses in call to 'print'